### PR TITLE
WAL-365

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: patches/@unimodules+core+7.1.2.patch [hunk 1]
```diff
@@ -1,319 +0,0 @@
-diff --git a/node_modules/@unimodules/core/.eslintrc.js b/node_modules/@unimodules/core/.eslintrc.js
```
Comment (hrehman-sphereon) [inline patches/@unimodules+core+7.1.2.patch:1]: For my info:
Why is this removed?
What was it doing?
Why isn't it required anymore?

File: .eslintrc.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     "plugin:import/typescript",
     "prettier"
   ],
-  "globals": { "BigInt": true, "console": true, "WebAssembly": true },
+  "globals": { "BigInt": true, "console": true, "WebAssembly": true, "__DEV__": true, "localStorage": true },
```
[the remark this answers is not in the document]
Comment (BtencateSphereon) [reply .eslintrc.json:15]: i'm not sure bout that, as this solves an issue about eslint complaining about the shim. What is your reason this should be false?

File: patches/@unimodules+core+7.1.2.patch (hunk 1 above)
Comment (BtencateSphereon) [reply patches/@unimodules+core+7.1.2.patch:1]: because the package has been removed

File: .eslintrc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (hrehman-sphereon) [reply .eslintrc.json:15]: Got it. I missunderstood.

File: yarn.lock [hunk 3]
```diff
@@ -3834,6 +3834,21 @@
     "@typescript-eslint/types" "5.45.1"
     eslint-visitor-keys "^3.3.0"
 
+"@unimodules/core@*":
```
Comment (hrehman-sphereon) [inline yarn.lock:3837]: As per comment referred by link ... Do we need to remove it from here then ? 

https://github.com/Sphereon-Opensource/ssi-mobile-wallet/pull/23#discussion_r1087732851

File: patches/@unimodules+core+7.1.2.patch (hunk 1 above)
Comment (nklomp) [reply patches/@unimodules+core+7.1.2.patch:1]: I am not too sure about this. Indeed it is still showing up in the yarn.lock file I see nothing that removes the package. It is being listed as an optional dep from isomorphic-webcrypto. 

So the question is. Where has this been explicitly removed and what is preventing its installation then?

Comment (BtencateSphereon) [reply patches/@unimodules+core+7.1.2.patch:1]: @nklomp this happened in sanders pull request

**Review by hrehman-sphereon — APPROVED** [review]

**Review by nklomp — APPROVED** [review]
